### PR TITLE
vapor 2 -> 3

### PR DESCRIPTION
**Pull request by tanner0101**

### CODE REVIEW

**Comment by waghanza** [comment]
Hi @tanner0101,

Thanks for updating `vapor`. 

However you are using a **branch**, not a **tag**.

I'll rather wait until `vapor` 3 (stable) is published :stuck_out_tongue:

**Comment by tanner0101** [comment]
@waghanza yes sorry should have mentioned that. The tag will be coming today, so wanted to have this PR ready 👍

**Comment by tanner0101** [comment]
@waghanza just updated to 3.0.0 official release. 👍

**Comment by waghanza** [comment]
@tanner0101 The `Dockerfile` seems to be **wrong**. 

Could you confirm that the _commit_ I have made is OK ?

File: swift/vapor/Dockerfile [hunk 1]
```diff
@@ -6,4 +6,4 @@ COPY Package.swift main.swift ./
 
 RUN swift build -c release
 
-CMD .build/release/server serve -e prod -h 0.0.0.0 -p 3000
+CMD .build/release/server serve --hostname 0.0.0.0 --port 3000 --env prod
```
Comment (tanner0101) [inline swift/vapor/Dockerfile]: oops, that should have been `-H` (capital). The change you made works, too. Thanks!

Comment (tanner0101) [reply swift/vapor/Dockerfile]: FWIW, you could write this as:
```
CMD .build/release/server serve -b 0.0.0.0:3000 -e prod
```

**Pull request merged**